x-pack/plugins/security_solution/common/endpoint/schema/policy.ts

```diff
@@ -19,3 +20,21 @@ export const GetAgentPolicySummaryRequestSchema = {
     policy_id: schema.nullable(schema.string()),
   }),
 };
+
+const ListWithKuerySchema = schema.object({
+  page: schema.maybe(schema.number({ defaultValue: 1 })),
+  perPage: schema.maybe(schema.number({ defaultValue: 20 })),
```

Comment:
can we rename this to pageSize for consistency https://github.com/elastic/kibana/blob/3d0bc8b70c226f4da428b475de901bf37746c0cc/x-pack/plugins/security_solution/common/endpoint/schema/metadata.ts#L16
https://github.com/elastic/kibana/blob/3d0bc8b70c226f4da428b475de901bf37746c0cc/x-pack/plugins/security_solution/common/endpoint/types/index.ts#L1241